rest-server/bin/engines.py

```diff
@@ -47,6 +47,18 @@
 #    except HttpError as err:
 #        print_failure(err, 'Error updating engine.')
 
+elif args.action == 'import':
+    engine_id = args.engine_id
+    print("Input path: {}".format(args.input))
+    try:
+        #  def update(self, engine_id, input_path=None, update_config=False, data={}):
+        res = engine_client.update(engine_id=engine_id, input_path=args.input, update_config=False, data={})
+        print_success(res, 'Importing to engine: {}\n'.format(engine_id))
+    except HttpError as err:
+        print_failure(err, 'Error importing to engine-id: {} from {}\n'.format(engine_id, args.input))
+    #  else:
+    #      print_failure(None, "Error: no input for import command.")
+
 elif args.action == 'delete':
```

Comment:
this seems to work ok, but the exception is always throw:

Error importing to engine-id: test_cb from /Users/pat/big-data/harness/test_cb_backup
 Request entity expected but not supplied

I'm not sure what is throwing the exception originally, the server or the python SDK?